src/App.js

```diff
@@ -10,6 +11,7 @@ class App extends Component {
       return (
         <div>
           <Nav />
+          <Status />
```

Comment:
We have a status bar that shows 'error' and 'saved'. But we don't have a thing which greys out the Publish button or shows a grey 'submitting' thing anywhere yet?